### ISSUE TITLE
Linux版本不更新了吗？

### ISSUE THREAD

**Issue by Jink92** [xpf0000/FlyEnv#147]
Linux版本不更新了吗？

**Comment by xpf0000** [comment]
目前完全是为爱发电. 时间精力不够.  暂无更新计划

**Comment by kalligator** [comment]
Is it the binaries only that are left behind? I.e. if we do the build ourselves will it be up to date?

**Comment by xpf0000** [comment]
NO. The Linux branch code is not the latest. macOS/Windwos/Linux have different operating mechanisms and cannot share a set of codes. They are developed separately.
